django-stubs/core/checks/messages.pyi

```diff
@@ -19,16 +19,16 @@ class CheckMessage:
     def is_silenced(self) -> bool: ...
 
 class Debug(CheckMessage):
-    def __init__(self, *args: Any, **kwargs: Any) -> None: ...
+    def __init__(self, msg: str, hint: Optional[str] = ..., obj: Any = ..., id: Optional[str] = ...) -> None: ...
```

Comment:
`level` is covered by the first call argument 👍 
https://github.com/django/django/blob/e30d6678421b7573a1995f61521f14348c9b2a17/django/core/checks/messages.py#L10